webpack/assets/javascripts/react_app/components/common/CloseButton.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import Icon from './Icon';
+
+const CloseButton = ({onClick}) => {
+  return (
+    <button className="close" aria-hidden="true" onClick={onClick}>
```

Comment:
clicking it is dispatching an action which changes the state of the notification to visible: false, and then it doesn't get rendered